libkbfs/md_journal.go

```diff
@@ -20,23 +20,21 @@ import (
 )
 
 // ImmutableBareRootMetadata is a thin wrapper around a
-// *BareRootMetadata that takes ownership of it and does not ever
-// modify it again. Thus, its MdID can be calculated and
-// stored. ImmutableBareRootMetadata objects can be assumed to never
-// alias a (modifiable) *BareRootMetadata.
+// ImmutableBareRootMetadata used to also cache the MdID and
```

Comment:
Any reason for getting rid of the old comment? I think it's still applicable, except `*BareRootMetadata` -> `BareRootMetadata`, and mentioning the timestamp